File: app/src/main/java/com/kota/Bahamut/MyApplication.java
package com.kota.Bahamut;

import android.app.Application;
import android.webkit.WebView;

public class MyApplication extends Application {
    static MyApplication _instance;

    public MyApplication() {
        _instance = this;
    }

    @Override
    public void onCreate() {
        super.onCreate();
        
        // 初始化WebView，避免多次初始化導致的錯誤
        try {
            WebView.setWebContentsDebuggingEnabled(false);
        } catch (Exception e) {
            e.printStackTrace();
        }
    }

    public static MyApplication getInstance() {
        return _instance;
    }
}

File: app/src/main/java/com/kota/Bahamut/MyApplication.kt
package com.kota.Bahamut

import android.app.Application
import android.webkit.WebView
import com.google.android.gms.games.PlayGamesSdk

class MyApplication : Application() {

    override fun onCreate() {
        super.onCreate()
        instance = this

        // 初始化 Google Play Games SDK
        PlayGamesSdk.initialize(this)

        // 初始化WebView，避免多次初始化導致的錯誤
        try {
            WebView.setWebContentsDebuggingEnabled(false)
        } catch (e: Exception) {
            e.printStackTrace()
        }
    }

    companion object {
        lateinit var instance: MyApplication
            private set
    }
}
